### PR TITLE
Webpackify React Create Element Lab

### PR DESCRIPTION
**Pull request by Lukeghenco**
@gj this is good to go. If you could clone this down and just make sure the build process and test suite runs. I also added a solution branch that works for this too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +23,27 @@ Your task is to create a React element that represents _you_ (or a close approxi
...
+
+To run the test suite type this in the terminal: 
+
+```
+npm test
+```
+
 Feel free to render out the component in the DOM using `ReactDOM.render()`.
 
+## NOTE!!
```
Comment (gj) [inline README.md:41]: Instead of this note, what do you think about creating a little aside here to introduce the idea of using unique keys in React?

Comment (Lukeghenco) [reply README.md:41]: I like that idea, but.. we go over it later, and I'm not sure we want to do that kind of logic in the React.createElement() function.

Comment (gj) [reply README.md:41]: Totally fair — maybe just a little teaser like "don't worry too much about this now — we'll go over it in an upcoming lesson" ?

Comment (Lukeghenco) [reply README.md:41]: Done, good catch 👍

Comment (Lukeghenco) [reply README.md:41]: That sounds good

**Pull request merged**